packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts

```diff
@@ -541,6 +543,12 @@ export class ContainerDefinition extends Construct {
       }
     }
 
+    if (props.dockerLabels) {
+      this.dockerLabels = { ...props.dockerLabels };
+    } else {
+      this.dockerLabels = {};
+    }
```

Comment:
```suggestion
    this.dockerLabels = { ...props.dockerLabels };
```
Am I missing something? 🤔